bin/development-server.js

```diff
@@ -67,13 +65,11 @@ app.get("/get", function(req, res) {
     body => res.json(JSON.parse(body))
   );
 
-  httpReq.on('error', function (err) {
-    res.status(500).send(err.code);
-  });
-})
+  httpReq.on("error", err => res.status(500).send(err.code));
+});
 
 // Listen
-app.listen(8000, "localhost", function(err, result) {
+app.listen(8000, "0.0.0.0", function(err, result) {
```

Comment:
This is the change that brought me to this file.  I'm testing things out with Edge and needed to be able to access `http://clarkbw.local:8000`.  I don't think there are security concerns here.  You can omit the entry altogether and have a similar effect bu this seems more straightforward.